jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlConfiguration.java

```diff
@@ -493,21 +501,6 @@ public Object configure() throws Exception
          */
         public void configure(Object obj, XmlParser.Node cfg, int i) throws Exception
```

Comment:
Something weird has gone on in the history!  I thought the idea of passing in the index was so that leading ignored Args could be skipped!